cmd/address_gen/address_gen.go

```diff
@@ -24,6 +25,8 @@ func main() {
 	genCount := flag.Int("n", 1, "Number of addresses to generate")
 	hideSecKey := flag.Bool("s", false, "Hide the secret key from the output")
 	isBitcoin := flag.Bool("b", false, "Print address as a bitcoin address")
+	hexSeed := flag.Bool("x", false, "Use hex string as seed")
+	verbose := flag.Bool("v", false, "Show verbose info of generated addresses")
```

Comment:
Change `verbose` to `j,json`?

Reply:
It's for showing details of generated address info, including pubkey, seckey, seed, etc. So I think verbose is ok.